mautic/mautic
# Elastic Email - add support for transactional emails

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Documentations to this request https://elasticemail.com/support/guides/unsubscribe/ (part Transactional Unsubscribe)
This PR set flag IsTransactional to True for all no segments email.


[//]: # ( As applicable: )

#### Steps to test this PR:
1. Send email 
2. Check header with IsTransactional in your email client https://mxtoolbox.com/Public/Content/EmailHeaders/
